sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/models/AzureCliToken.java

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.implementation.models;
+
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A wrapper class for deserializing a token payload returned from the Azure CLI.
+ */
+public class AzureCliToken implements JsonSerializable<AzureCliToken> {
```

Comment:
Just a few nits on using standard patterns by making this final and using set and get in property accessors.